### PR TITLE
updated faq and submit pages

### PR DESCRIPTION
**Pull request by whatcoloris**
requesting a pull!

### CODE REVIEW

File: _submit/submission-FAQ.md [hunk 1]
```diff
@@ -8,50 +8,56 @@ banner: true
...
 
   <dt>Q: What kind of image should I submit?</dt>
-  <dd>A: Though photos are preferable, any image (a photo, a drawing, a diagram, etc.) is acceptable if you think it is relevant to the work. Images will be printed in black and white, so please submit them in grayscale. Images must be 5×7 300dpi tiff files and may be in portrait or landscape orientation. Please credit the artist or photographer when you send us the image, and make sure you have permission/copyright to publish it.</dd>
+  <dd>A: Though photos are preferable, any image (a photo, a drawing, a diagram, etc.) is acceptable if you think it is relevant to the work. Images will be printed in black and white, so please submit them in grayscale. Images must be 5×7 300dpi tiff files and may be in portrait or landscape orientation. Please credit the artist or photographer when you send us the image, and make sure you have permission/copyright to publish it. Please send your image when you complete the submission form.</dd>
```
Comment (edwardsharp) [inline _submit/submission-FAQ.md]:
```suggestion
  <dd>A: Though photos are preferable, any image (a photo, a drawing, a diagram, etc.) is acceptable if you think it is relevant to the work. Images will be printed in black and white, so please submit them in grayscale. Images must be 5×7 300dpi tiff files and may be in portrait or landscape orientation. Please credit the artist or photographer when you send us the image, and make sure you have permission/copyright to publish it. Please include your image with your submission form.</dd>
```

hmm, i feel like this is more specific?

**Review by edwardsharp — COMMENTED** [review]
just one little thing 🤔

File: _submit/submission-FAQ.md (hunk 1 above)
Comment (whatcoloris) [reply _submit/submission-FAQ.md]: OH yeah I think that's great.

**Review by edwardsharp — APPROVED** [review]
💎 ty! 😻

**Pull request merged**